src/main/java/net/snowflake/ingest/streaming/internal/DataValidationUtil.java

```diff
@@ -70,6 +76,8 @@ class DataValidationUtil {
 
   private static final ObjectMapper objectMapper = new ObjectMapper();
 
+  private static final JsonFactory factory = new JsonFactory();
```

Comment:
the objectmapper is configured in a static ctor for a couple data types, does this factory object not need a similar configuration?

Reply:
It does not, the configured serializers only serialize custom Java data types, which we don't have when we work with JSON provided as string.